examples/stencil-kitchen-sink/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "stencil-kitchen-sink",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Stencil Component Starter",
+  "license": "ISC",
+  "author": "",
+  "main": "dist/index.cjs.js",
+  "unpkg": "dist/stencil-kitchen-sink/stencil-kitchen-sink.esm.js",
+  "module": "dist/custom-elements/index.js",
+  "types": "dist/custom-elements/index.d.ts",
+  "files": [
+    "dist/",
+    "loader/"
+  ],
+  "scripts": {
+    "build": "stencil build --docs",
+    "build.watch": "stencil build --docs --watch",
+    "start": "stencil build --dev --watch --serve",
+    "test": "stencil test --spec --e2e",
+    "test.watch": "stencil test --spec --e2e --watchAll",
+    "generate": "stencil generate",
+    "storybook": "start-storybook -p 9001"
+  },
+  "dependencies": {
+    "@stencil/core": "^2.0.1"
+  },
+  "devDependencies": {
+    "@storybook/source-loader": "*",
+    "@storybook/stencil": "../../app/stencil",
```

Comment:
This will actually pack that folder into the cache, you should use a version instead